pyanaconda/installation.py

```diff
@@ -176,9 +176,19 @@ def run_generate_initramfs():
         # Join a realm. This can run only after network is configured in the target system chroot.
         configuration_queue.append_dbus_tasks(SECURITY, [security_proxy.JoinRealmWithTask()])
 
-    post_scripts = TaskQueue("Post installation scripts", N_("Running post-installation scripts"))
-    post_scripts.append(Task("Run post installation scripts", runPostScripts, (ksdata.scripts,)))
-    configuration_queue.append(post_scripts)
+    from pyanaconda import product
+
+    if product.productName != "RHVH":
+        post_scripts = TaskQueue(
+            "Post installation scripts",
+            N_("Running post-installation scripts")
+        )
+        post_scripts.append(Task(
+            "Run post installation scripts",
+            runPostScripts,
+            (ksdata.scripts,)
+        ))
+        configuration_queue.append(post_scripts)
```

Comment:
If there's no special reason to write the ks earlier, maybe we can just have it done later and not differentiate between RHVH and others.